crates/local-cluster-runner/src/node/mod.rs

```diff
@@ -356,7 +348,23 @@ impl Node {
 
         let mut child = cmd.spawn().map_err(NodeStartError::SpawnError)?;
         let pid = child.id().expect("child to have a pid");
+
+        let admin_address = if self.config().has_role(Role::Admin) {
+            Cow::Owned(self.config().admin.bind_address.to_string())
+        } else {
+            Cow::Borrowed("None")
+        };
+
+        let ingress_address = if self.config().has_role(Role::Worker) {
+            Cow::Owned(self.config().ingress.bind_address.to_string())
+        } else {
+            Cow::Borrowed("None")
+        };
+
         info!(
```

Comment:
It would be neat to also log the command ready to copy-paste, something like:

```rs
info!("To connect using restate CLI:\nexport RESTATE_ADMIN_URL=http://{}", %admin_address);
```

Reply:
Good idea. Will add it.